pom.xml

```diff
@@ -38,17 +38,18 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.60</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
🐛 Hard dependency on Jenkins 2.60 and JDK 8 means we can no longer bundle this version in Blue Ocean's aggregator :(

Is this a necessary change?